src/java.base/share/classes/java/util/random/package-info.java

```diff
@@ -0,0 +1,469 @@
+ * invoke methods of {@code g} such as
+ * {@link RandomGenerator#nextLong nextLong()},
+ * {@link RandomGenerator#nextInt nextInt()},
+ * {@link RandomGenerator#nextFloat nextFloat()},
+ * {@link RandomGenerator#nextDouble nextDouble()} and
+ * {@link RandomGenerator#nextBoolean nextBoolean()} to generate individual
+ * randomly chosen values. One can also use the methods
+ * {@link RandomGenerator#ints ints()}, {@link RandomGenerator#longs longs()}
+ * and {@link RandomGenerator#doubles doubles()} to create streams of randomly
+ * chosen values. The methods
+ * {@link RandomGenerator#nextGaussian nextGaussian()} and
+ * {@link RandomGenerator#nextExponential nextExponential()} draw floating-point
+ * values from nonuniform distributions. The method
+ * {@link RandomGenerator#period period()} returns information about the period
+ * of the generator instance.
+ *
+ * <p> For a multi-threaded application, one can repeat the preceding steps to
+ * create additional {@linkplain RandomGenerator RandomGenerators}, but often it is preferable
+ * to use methods of the one single initially created generator to create others
+ * like it. (One reason is that some generator algorithms, if asked to create a
+ * new set of generators all at once, can make a special effort to ensure that
+ * the new generators are statistically independent.) If the initial generator
+ * implements the interface {@link StreamableGenerator}, then the method
+ * {@link RandomGenerator#rngs rngs()} can be used to create a stream of
+ * generators. If this is a parallel stream, then it is easy to get parallel
+ * execution by using the {@link Stream#map map()} method on the stream.
+ *
+ * <p> For a multi-threaded application that forks new threads dynamically,
+ * another approach is to use an initial generator that implements the interface
+ * {@link RandomGenerator.SplittableGenerator}, which is then considered to
+ * "belong" to the initial thread for its exclusive use; then whenever any
+ * thread needs to fork a new thread, it first uses the
+ * {@link RandomGenerator#split split()} method of its own generator to create a
+ * new generator, which is then passed to the newly created thread for exclusive
+ * use by that new thread.
+ *
+ *
+ * <h2>Choosing a Random Number Generator Algorithm</h2>
+ *
+ * If an application requires a random number generator algorithm that is
+ * cryptographically secure, then it should use an instance of the class
+ * {@link java.security.SecureRandom}.
+ *
+ * <p> For applications (such as physical simulation, machine learning, and
+ * games) that do not require a cryptographically secure algorithm, this package
+ * provides multiple implementations of interface {@link RandomGenerator} that
+ * provide trade-offs among speed, space, period, accidental correlation, and
+ * equidistribution properties.
+ *
+ * <p> For applications with no special requirements,
+ * "L64X128MixRandom" has a good balance among speed, space,
```

Comment:
The documentation assumes that the `jdk.random` module is present in the JDK image. Perhaps we need to spit the specifics to `jdk.random`?

Reply:
But jdk.random isn't really a public API.